engine/schema/src/main/resources/META-INF/db/schema-41720to41800.sql

```diff
@@ -997,3 +997,9 @@ BEGIN
     DECLARE CONTINUE HANDLER FOR 1061 BEGIN END; SET @ddl = CONCAT('ALTER TABLE ', in_table_name); SET @ddl = CONCAT(@ddl, ' ', ' ADD KEY ') ; SET @ddl = CONCAT(@ddl, ' ', in_index_name); SET @ddl = CONCAT(@ddl, ' ', in_key_definition); PREPARE stmt FROM @ddl; EXECUTE stmt; DEALLOCATE PREPARE stmt; END;
 
 CALL `cloud`.`IDEMPOTENT_ADD_KEY`('i_user_ip_address_state','user_ip_address', '(state)');
+
+CALL ADD_GUEST_OS_AND_HYPERVISOR_MAPPING (1, 'AlmaLinux 9.0', 'KVM', 'default', 'AlmaLinux 9.0');
```

Comment:
Leave it just 9, so in future we need not add for all 9.0, 9.1, 9.2…. Also should we find and add guest os mappings for other hypervisors? (Could be a separate issue or PR)